modules/claims_api/app/controllers/claims_api/v2/veterans/intent_to_file_controller.rb

```diff
@@ -133,7 +142,7 @@ def claimant_ssn_equals_vet_ssn?(options)
         end
 
         def get_bgs_type(params)
-          itf_types[params[:type].downcase]
+          params[:data] ? itf_types[params[:data][:attributes][:type].downcase] : itf_types[params[:type].downcase]
```

Comment:
^ this is possible. if we don't include attributes and only include type (ex: `{ data: { type: '' }}`), it will throw this `nil` error. i think we could prevent this in a few ways, but maybe updating `validate_request_format` is the easiest approach?